src/plans/D3MCompoundV2TypeRateTargetPlan.sol

```diff
@@ -46,7 +46,7 @@ pragma solidity ^0.8.14;
 
 import "./ID3MPlan.sol";
 
-// cDai - https://etherscan.io/address/0x5d3a536E4D6DbD6114cc1Ead35777bAB948E3643
+// cDai - https://github.com/compound-finance/compound-protocol/blob/master/contracts/CErc20.sol
```

Comment:
Personally I would use a commit hash rather than master here.

Raw link:
https://raw.githubusercontent.com/compound-finance/compound-protocol/a3214f67b73310d547e00fc578e8355911c9d376/contracts/CErc20.sol

Github link:
https://github.com/compound-finance/compound-protocol/blob/a3214f67b73310d547e00fc578e8355911c9d376/contracts/CErc20.sol

Reply:
If the concern is that this cDai address may be deprecated in the future then we can stick to the method you've used here, but it isn't consistent with how we link to github stuff for things like execs in spells. Just my 2c

Reply:
Yeah I was thinking on doing the commit hash as well, but didn't have the time to go over one by one to make extremely sure it was all perfectly matching. So left it as a soft reference.

We have a few options:
1) Leave as it is
2) Put a comment making explicit is a soft reference
3) Go contract by contract and check the implementation is exactly that and then hardening the commit (if you want to do that sounds great to me).

Reply:
I thought of this as well, and personally think it's fine to leave as a soft reference in this instance.

Reply:
We can create a new PR if this change wants to be pursue, opting for option 1 for now.

Reply:
This is fine then as long as we know what we're doing